docs/integrations/app-development/bitbucket.md

```diff
@@ -141,14 +140,9 @@ If you want to deployment events to multiple Sumo Logic orgs, include a `-pipe`
 
 For reference: This is how the [bitbucket-pipelines.yml](https://bitbucket.org/app-dev-sumo/backendservice/src/master/bitbucket-pipelines.yml) looks after adding deploy pipe code to our sample Bitbucket CI/CD pipeline.
 
-
 ### Step 4: Enable Bitbucket Event-Key tagging at Sumo Logic
 
-Sumo Logic needs to understand the event type for incoming events (for example, repo:push events). To enable this, the [X-Event-Key](https://confluence.atlassian.com/bitbucket/event-payloads-740262817.html#EventPayloads-HTTPheaders) event type needs to be enabled. To enable this, perform the following steps in the Sumo Logic console:
-
-1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Logs > Fields**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the top menu select **Configuration**, and then under **Logs** select **Fields**. You can also click the **Go To...** menu at the top of the screen and select **Fields**. 
-2. Add Field ‎**X-Event-Key**‎.<br/><img src={useBaseUrl('img/integrations/app-development/BB_Collect_Log.png')} alt="Bitbucket" />
-
+To properly identify the event type for incoming events (for example, repo:push events), Sumo Logic automatically adds the [X-Event-Key](https://confluence.atlassian.com/bitbucket/event-payloads-740262817.html#EventPayloads-HTTPheaders) event type to the [Fields](/docs/manage/fields) during app installation.
 
 ## Installing the Bitbucket App
 
```

Comment:
We also need to use reusable section of viewdashboard. like this : https://github.com/SumoLogic/sumologic-documentation/pull/5190/files#diff-e6ee37582db492e7686850f5ceff35579fc71b1a7af2d74ad37f73b1101fa05aR409

Reply:
Need to do the same for all apps as part of non-FER migrated apps - bitbucket,github,gitlab,jira cloud, opsgenie,pagerduty v2 and pagerduty v3